src/client/eppo-client.ts

```diff
@@ -131,40 +132,32 @@ export default class EppoClient {
   private assignmentCache?: AssignmentCache;
   // whether to suppress any errors and return default values instead
   private isGracefulFailureMode = true;
-  private expectObfuscated: boolean;
-  private obfuscationMismatchWarningIssued = false;
-  private configObfuscatedCache?: boolean;
   private requestPoller?: IPoller;
   private readonly evaluator = new Evaluator();
   private configurationRequestor?: ConfigurationRequestor;
   private readonly overrideValidator = new OverrideValidator();
 
   constructor({
     eventDispatcher = new NoOpEventDispatcher(),
-    isObfuscated = false,
+    isObfuscated,
```

Comment:
It looks like [client](https://github.com/Eppo-exp/js-client-sdk/blob/30d3a85189b216f92af6dcd67c852702fddaa12e/src/index.ts#L123) SDK is passing this option, so we need to pair this PR with client changes or users will see an unexpected deprecation warning

Reply:
👍 
https://github.com/Eppo-exp/js-client-sdk/pull/192
Needs the release cut, of course.